lib/twitter_cldr/version.rb

```diff
@@ -4,5 +4,5 @@
 # http://www.apache.org/licenses/LICENSE-2.0
 
 module TwitterCldr
-  VERSION = "3.1.2"
+  VERSION = "3.2.0"
```

Comment:
Did you mean patch? If so, patch is the last number and indicates a bug fix. Otherwise this is a minor version bump since it is an addition:

http://semver.org/